### PR TITLE
Testing/journal

### PR DESCRIPTION
**Pull request by alivaditis**
## What I did:

Cypress testing for journal.
Added happy paths/ sad path for no journal entries saved
Added some logic for preventing duplicates in collection

I could not for the life of me figure out how to access the star rating value because it's a weird component from a package, so that is the only thing I left out.

Also changed the data for user2 journal entries

Did this break anything?

- [x]  No
- [ ]  Yes

Type of change

- [x]  New feature (non-breaking change which adds functionality)
- [ ]  Bug fix (non-breaking change which fixes an issue)
- [x]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ]  Super small fix (Corrected a typo, removed a comment, etc.)
- [ ]  Skip all the other stuff and briefly explain the fix.

Checklist:

- [x]  If this code needs to be tested, all tests are passing
- [x]  The code runs locally
- [ ]  There are comments where clarification/ organization is needed
- [x]  The code is DRY. If it's not, I tried my best
- [x]  I reviewed my code before pushing

What's next?

### CODE REVIEW

File: src/components/Entry/Entry.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ function Entry ({id, title, artists, date, notes, image, rating, masterId}) {
         <div className='entry-info'>
           <p className='entry-title'>{title}</p>
           <p className='entry-artist'>{artists}</p>
-          {rating>0 && <Rating initialValue={rating} readonly={true} size='12'/>}
+          {rating>0 && <Rating className='rating' initialValue={rating} readonly={true} allowFraction={true} size='12'/>}
```
Comment (jmartin777) [inline src/components/Entry/Entry.js:33]: whats this line of code doing that you input, for clarity?

File: cypress/e2e/journal-spec.cy.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+
+describe('template spec', () => {
```
Comment (jmartin777) [inline cypress/e2e/journal-spec.cy.js:2]: clean test!

**Review by jmartin777 — APPROVED** [review]
Really great seeing the refactor in this test and how you set it up!

File: src/components/Entry/Entry.js (hunk 1 above)
Comment (alivaditis) [reply src/components/Entry/Entry.js:33]: it's the allowFraction={true}

I forgot to add it the first go around so it was only ever showing whole stars even when you submitted halves, this fixes  that

File: cypress/e2e/journal-spec.cy.js [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+    cy.fixture('mockUser2').as('user2')
+      .get('@user2').then((user) => {
+        cy.get('.username-field').type(user.username)
+        .get('.username-field').should('have.attr', 'value', 'user2')
+        .get('.password-field').type(user.password)
+        .get('.password-field').should('have.attr', 'value', 'sound-stash')
+        .get('form > .standard-btn').click()
+      })
+    
+    cy.wait('@getTrending')
```
Comment (jalbe0076) [inline cypress/e2e/journal-spec.cy.js:29]: don't need to wait here. It still get's stubbed on my end

File: cypress/e2e/journal-spec.cy.js [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+        cy.get('.username-field').type(user.username)
+        .get('.username-field').should('have.attr', 'value', 'user2')
+        .get('.password-field').type(user.password)
+        .get('.password-field').should('have.attr', 'value', 'sound-stash')
+        .get('form > .standard-btn').click()
+      })
+
+    cy.get('[href="/journal"]').click()
+      .url().should('include', '/journal')
+      .wait('@getTrending')
```
Comment (jalbe0076) [inline cypress/e2e/journal-spec.cy.js:55]: don't need to wait here. It still get's stubbed on my end

**Review by jalbe0076 — CHANGES_REQUESTED** [review]
Nice Testing, some minor lines are not needed.

**Pull request merged**